<launch>
  <!-- Map Server -->
  <node pkg="map_server" type="map_server" name="map_server"
        args="$(find path_planning)/maps/slam_maps/it2_last.yaml" />

  <!-- Central Path Planner -->
  <node pkg="path_planning" type="path_planner_node.py" name="path_planner" output="screen">
    <param name="grid_path" value="$(find path_planning)/maps/grid.npy" />
  </node>


  <!-- Robot 1 -->
  <group ns="robot_1">
    <param name="robot_description" textfile="$(find path_planning)/urdf/robot_1.urdf" />
    <node name="rsp1" pkg="robot_state_publisher" type="robot_state_publisher" output="screen"/>
  </group>

  <!-- Robot 2 -->
  <group ns="robot_2">
    <param name="robot_description" textfile="$(find path_planning)/urdf/robot_2.urdf" />
    <node name="rsp2" pkg="robot_state_publisher" type="robot_state_publisher" output="screen"/>
  </group>

  <!-- RViz -->
  <node name="rviz" pkg="rviz" type="rviz"
        args="-d $(find path_planning)/rviz/multi_robot_debug.rviz" output="screen"/>
</launch>
